docs/incompatible_changes/3.5.rst

```diff
@@ -0,0 +1,756 @@
+always merged instead of being *sometimes* overridden and *sometimes* merged.
+
+For example, this code has only checked for ``embed_links`` permission on 3.4
+but now checks for both ``add_reactions`` and ``embed_links`` permissions:
+
+.. code:: python
+
+    from redbot.core import commands
+
+    class MyCog(commands.Cog):
+        @commands.command()
+        @commands.bot_has_permissions(embed_links=True)
+        @commands.bot_has_permissions(add_reactions=True)
+        async def example(self, ctx):
+            msg = await ctx.send(embed=discord.Embed(description="Hello!"))
+            await msg.add_reaction("\N{SMILING FACE WITH OPEN MOUTH}")
+
+Note that stacking `@commands.has_permissions() <redbot.core.commands.has_permissions()>`
+with ``@commands.*_or_permissions()`` decorators still behaves differently depending on
+the decorator order.
+
+Some of the primary dependencies have been removed or replaced
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+While this may not technically fall under `our version guarantees <version_guarantees>`,
+we recognize that this may affect some people.
+As such, below we list the noteworthy dependency replacements and removals (package names
+are PyPI distribution names):
+
+- ``appdirs`` package has been replaced with ``platformdirs``
+- ``chardet`` package has been replaced with ``charset-normalizer``
+- ``fuzzywuzzy`` package has been replaced with ``rapidfuzz``
+- Red no longers depends on ``aiosqlite``, ``PyNaCl``, and ``python-Levenshtein-wheels``
+
+discord.py version has been updated to 2.2.3
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To allow Red to continue operating *and* use the newer features (threads, text in voice/stage channels,
+buttons, application commands, and AutoMod, to name a few), we've updated discord.py from version
+1.7.3 to 2.2.3. Since this is a major upgrade, it will require you to update your code accordingly.
+discord.py's documentation has :dpy_docs:`a migration guide <migrating>` that you can follow in order to
+update your cogs.
+
+Red itself has already been updated to support all of these changes that happened over the years.
+In order to do so, we've also had to make a few breaking changes of our own. Those changes are
+normal sections in the current document so be sure to follow it in full and you should be able to
+perform all the necessary changes.
+
+To help support some of the newer features, we've also added a few things:
+
+- Utilities that allow to **properly** check whether a member can do something in threads (or channels)
+  for cases where it isn't as simple as checking permissions:
+
+    - Command check decorators: `bot_can_manage_channel()`, `bot_can_react()`
+    - Permission check decorators: `can_manage_channel()`, `guildowner_or_can_manage_channel()`,
+      `admin_or_can_manage_channel()`, `mod_or_can_manage_channel()`
+    - Functions: `can_user_send_messages_in()`, `can_user_manage_channel()`, `can_user_react_in()`
```

Comment:
Is this intended to be indented? The comment above it specifies a "few things" were added, but there is only 1 top level bullet point

Reply:
The comment above was rephrased a few times as I was trying to decide what I'll be listing there and I guess it just ended up with only a single thing that I didn't have a better place for mentioning. Do you know if there's anything else that we could possibly mention here? If not then it might make sense to rephrase.

Reply:
`SimpleMenu` / `SetApiModal` / `SetApiView`?

Reply:
Added that.